pages/profile/invitations/index.js

```diff
@@ -25,5 +28,27 @@ module.exports = settings => {
     }
   })({ schema }));
 
+  app.post('/:invitationId/:action', (req, res, next) => {
+    const allowedActions = ['delete', 'resend', 'cancel'];
+
+    if (!allowedActions.includes(req.params.action)) {
+      return next(new BadRequestError('Unrecognised action'));
+    }
+
+    const { invitationId, action } = req.params;
+    const isDeletion = action === 'delete';
+    const params = {
+      method: isDeletion ? 'DELETE' : 'PUT'
+    };
+    const suffix = isDeletion ? '' : `/${action}`;
```

Comment:
```suggestion
    const suffix = isDeletion ? '' : `/delete`;
```

We've just asserted that `action === 'delete'` so might as well be explicit about what's going on.

Reply:
that doesn't make any sense :(

Reply:
if its a DELETE we _dont_ want a suffix

Reply:
we just DELETE /:id

Reply:
My bad.